lib/smart_answer_flows/marriage-abroad/outcomes/_link_to_consular_fees.govspeak.erb

```diff
@@ -1,3 +0,0 @@
-<% if calculator.services.any? %>
-  You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for <%= calculator.country_name_lowercase_prefix %>](/government/publications/<%= calculator.ceremony_country %>-consular-fees).
-<% end %>
```

Comment:
I've introduced an earlier commit (subject "DRY up condition around rendering link to consular fees") that removes the `if calculator.services.any?` statement from this partial.